mlflow/models/python_api.py

```diff
@@ -113,6 +117,10 @@ def predict(
         model_uri: URI to the model. A local path, a local or remote URI e.g. runs:/, s3://.
         input_data: Input data for prediction. Must be valid input for the PyFunc model. Refer
             to the :py:func:`mlflow.pyfunc.PyFuncModel.predict()` for the supported input types.
+
+            .. note::
+                Use `mlflow.models.convert_input_example_to_serving_input` to manually validate
```

Comment:
Could you elaborate when to use `convert_input_example_to_serving_input`? "manually validate" is a bit vague because this API itself is also a sort of manual validation😅